### ISSUE TITLE
order parcles

### ISSUE THREAD

**Issue by tongalalal** [allegro/allegro-api#9708]
Here I use this interface https://api.{environment}/order/checkout-forms/{id}/shipments to obtain the shipment number data
{"shipments":[{"waybill":"0000007268757Q","carrierId":"DPD","lineItems":[{"id":"6f8d2600-6031-11ef-bf5b-2798c64ba73d"}], "createdAt":"2024-08-22T03:48:44Z","id":"RFBEOjAwMDAwMDcyNjg3NTdR"}]} Then I want to use this number data to request https://api.{environment}/shipme for this interface nt-management/shipments/{shipmentId} Package details but it shows no find why, This is the error message {"errors":[{"code":"SHIPMENT_NOT_FOUND","message":"Requested shipment was not found","details":"Requested shipment  was not found","path":"shipmentId","userMessage":"Nie odnaleziono przewozu","metadata":{}}]}

**Comment by MaciejFrackowiak** [comment]
It doesn't work like that. First, you have to generate a shipment by [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation#operation/createNewShipment) and then we will add a waybill to the order details.

**Comment by tongalalal** [comment]
@MaciejFrackowiak I have a doubt here, where do I get these request data to create the package? Are these interfaces able to return, or can they only be entered manually
![image](https://github.com/user-attachments/assets/f19025c1-fed4-4320-bf5e-e89c7bd803a1)

**Comment by MaciejFrackowiak** [comment]
You must provide them manually based on data retrieved from the selected order.

**Comment by tongalalal** [comment]
@MaciejFrackowiak 1. When I created the shipment, the hourly delivery method is incorrect, where can I see the correct delivery method of the order, here it shows that I have an error ac0359a2-900f-4f13-b00a-9ee747120d34, which is the commandId of the error.
2. When I create a new package, I don't seem to see any request field that identifies my order, so if I request to create a shipment, how do I know which order was created for me?

**Comment by MaciejFrackowiak** [comment]
The correct delivery method you can find in "delivery.method.id" field in [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation/#operation/getOrdersDetailsUsingGET).
We recognize orders/shipments by order email address.

**Comment by tongalalal** [comment]
@MaciejFrackowiak 
1、"commandId": "9873b525-ceb4-420d-b581-3682bc51184d","message": "Cannot create a shipment using own contract. Delivery method assigned to this order is the Allegro Standard one. Change the service and try again.","details": "Delivery Method is not available for this order",
2、commandId": "c4bbdbd1-59cf-427b-b980-5c297819ae7c
"message": "Cannot create a shipment using the Allegro Standard contract. Delivery method assigned to this order is the merchant controlled one. Change the service and try again.",
  "details": "Delivery Method is not available for this order",
  Please help me see what the two errors are of mine, I did use the fields in the delivery.method.id to create the shipment, but when I checked the creation status, these are the errors

**Comment by MaciejFrackowiak** [comment]
You changed the original delivery method specified in the order in both cases. In the order assigned to the email address you provided in the shipments, there was a delivery method with id="0d021121-97e5-4729-9af6-a05779c6b835" -  but you changed it. Every single order has its delivery method and you cannot change it.

**Comment by tongalalal** [comment]
@MaciejFrackowiak Here I thought that I changed the delivery method of the order, but after I used my buyer's account to place an order again, I asked again, and the error is still displayed here, {
"commandId": "511e07b7-c873-430e-9cc3-cf771049ebc3",
"status": "ERROR",
"errors": [
{
"code": "DELIVERY_METHOD_NOT_AVAILABLE",
"message":  "Cannot create a shipment using the Allegro Standard contract. Delivery method assigned to this order is the merchant  controlled one. Change the service and try again.",
"details": "Delivery Method is not available for this order",
"path": "deliveryMethodId",
"userMessage":  "Nie mo eh na utworzy przesy? ki przy u eh yciu umowy Allegro Standard. Metoda przypisana do tego zamowienia to umowa w? asna. "Nie Mo eh na utworzy u eh yciu umowy allegro standard. Metoda Przypisana do tego zamowienia to umowa w? Asna. Zmie UN umow pro i sprobuj ponownie."
}
].
"shipmentId": null
}
I did not modify the delivery method in my seller's background this time, and I am not quite sure which link is wrong, please help me to check it. This is my order number: 4a2afb30-6130-11ef-b716-654332907af4

**Comment by MaciejFrackowiak** [comment]
Unfortunately, you mixed up the order data. You provided an email address from a different order, so the assigned delivery method is not the same as you used during shipment creation.

**Comment by tongalalal** [comment]
@MaciejFrackowiak {
    "commandId": "694b4cd7-03d1-40a3-bad9-bcfeaadbd625",
    "status": "ERROR",
    "errors": [
        {
            "code": "CARRIER_CREDENTIALS_NOT_FOUND",
            "message": "Requested carrier credentials was not found",
            "details": "Requested carrier credentials was not found",
            "path": "credentialsId",
            "userMessage": "Wybrana umowa nie została odnaleziona"
        }
    ],
    "shipmentId": null
}，What about this error, I think I have requested the shipment with the correct email, but I did get this error and I still can't create a package for my order

**Comment by MaciejFrackowiak** [comment]
Yes, the email was correct, as well as the delivery method. However, in the field "credentialsId" you provided a value "null" - that was incorrect. In cases where you do not use "credentialsId" you should pass just null or please skip this field.

**Comment by tongalalal** [comment]
@MaciejFrackowiak {
    "commandId": "74bf3b79-19c4-4033-9a64-1922fbdffd1f",
    "status": "ERROR",
    "errors": [
        {
            "code": "CREDENTIALS_ID_NOT_PROVIDED",
            "message": "卖方没有提供自己的交货方式的凭证",
            "details": "Seller didnt provide credentials for own delivery Method",
            "path": "credentialsId",
            "userMessage": "Wybrana metoda dostawy nie obsługuje tego zamówienia"
        }
    ],
    "shipmentId": null
}Where can I provide this proof of delivery?

**Comment by MaciejFrackowiak** [comment]
Test data you should receive from the courier company DPD and then provide its details to [Ship with Allegro settings](https://allegro.pl.allegrosandbox.pl/moje-allegro/sprzedaz/zamowienia/ustawienia).

**Comment by tongalalal** [comment]
@MaciejFrackowiak 1、Here I am the test data, I don't have this tax identification number, what should I do, I just need a test package, but it has not been generated
![image](https://github.com/user-attachments/assets/b03f0900-67b3-4ad0-9ccd-df3acd75a9a9)
2、Should I provide this data, and where can I get the data for these tests, and set up this DPD own agreement
![image](https://github.com/user-attachments/assets/dc4ea8a8-df76-45c3-bb0c-bd209ad88e5b)

**Comment by MaciejFrackowiak** [comment]
All of these data you should receive from InPost and DPD. If you have any questions,  contact them directly.
